server/views/pages/induction/previousWorkExperience/workExperienceDetail.njk

```diff
@@ -50,7 +50,7 @@ Data supplied to this template:
                 attributes: { "aria-live": "polite" }
               },
               attributes: { "aria-label" : "Give the Job role" },
-              errorMessage: errors | findErrors('jobRole')
+              errorMessage: errors | findError('jobRole')
```

Comment:
Whoops! Didnt follow my own confluence page to the letter when I first committed this! 😊